portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/structure/WebContentWithCustomStructures.testcase

```diff
@@ -2128,6 +2128,16 @@ definition {
 		WebContent.viewTitle(webContentTitle = "WC WebContent Title");
 	}
 
+	@description = "This is a test for LPS-125933. Create, read, update and delete a Web Content from a Structure with all possible fields and using a Template with all posible fields."
+	@ignore = "true"
+	@priority = "5"
+	test ViewAndEditAsWellAsDeleteContentWithTitleAndAllFields {
+		property portal.acceptance = "false";
```

Comment:
Maybe we can add these P5 issues to acceptance. What do you think?

Reply:
let's wait the case to be stable and change that, what do you think?

Reply:
makes sense!